.github/workflows/verify-ecosystem-files.yaml

```
@@ -0,0 +1,71 @@
+name: Verify Ecosystem Upgrade Files
+permissions:
+  contents: read
+
+on:
+  pull_request:
+    paths:
+      - 'data/**'
+
+jobs:
+  verify-files:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Verify ecosystem files against era-contracts
+        run: |
+          set -e
+        
+          # NOTE: This is the branch of era-contracts that we consider as our source of truth
+          # This branch should be set to the development branch, eg: "draft-v29", or "draft-v30"
+          ERA_CONTRACTS_BRANCH="main"
+          ERA_CONTRACTS_REPO="matter-labs/era-contracts"
+          ERA_CONTRACTS_DIR="era-contracts"
+
+          echo "Checking out ${ERA_CONTRACTS_REPO} at branch ${ERA_CONTRACTS_BRANCH}"
+          git clone --depth 1 --branch "$ERA_CONTRACTS_BRANCH" "https://github.com/${ERA_CONTRACTS_REPO}.git" "$ERA_CONTRACTS_DIR"
+
+          # Find all ecosystem files in the data directory
+          FILES=$(find data -type f -name "*.yaml")
```

I think we should support`.yml` too, even if it is not used today. 
```suggestion
          FILES=$(find data -type f \( -name "*.yaml" -o -name "*.yml" \))
```